contract-tests/sdkClientEntity.js

```diff
@@ -9,6 +9,7 @@ export { badCommandError };
 export function makeSdkConfig(options, tag) {
   const cf = {
     logger: sdkLogger(tag),
+    diagnosticOptOut: true
   };
   const maybeTime = (seconds) =>
     seconds === undefined || seconds === null ? undefined : seconds / 1000;
```

Comment:
Unrelated, but this can be simplified:

```suggestion
    !seconds ? undefined : seconds / 1000;
```

Reply:
0 seconds is falsey.

Reply:
```
> let seconds = 0;
undefined
> !seconds ? undefined : seconds/1000
undefined
> seconds === undefined || seconds === null ? undefined : seconds/1000
0
```

I do not know if our test execution does any tests where it sets to 0, but if it does, then the results would be substantially different. As undefined will get default times, and 0 may or may not depending on the check.

Reply:
Now that we are post 8.0 I should convert this to typescript and I could make things like this at least look a little cleaner. This is still the original implementation from Eli, because I wanted to validate compatibility originally.